src/main/java/mate/academy/spring/config/DataInitializer.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.spring.config;
+
+import java.util.Set;
+import javax.annotation.PostConstruct;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.RoleService;
+import mate.academy.spring.service.UserService;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class DataInitializer {
+    private final RoleService roleService;
+    private final UserService userService;
+
+    public DataInitializer(RoleService roleService, UserService userService) {
+        this.roleService = roleService;
+        this.userService = userService;
+    }
+
+    @PostConstruct
+    public void inject() {
+        Role adminRole = new Role();
+        adminRole.setRoleName(Role.RoleName.ADMIN);

```

Comment:
Please try to add static import of Role.Rolename.ADMIN and USER
Then you'll be able to use them without classes prefixes and code will look more tidy